test11.js

```diff
@@ -0,0 +1,10 @@
+var foo = '' ;
+
+if(foo  === 'bar') {
+  console.log( 'hey' )
+}
+
+
+if(foca  === 'foca') {
+  console.log( 'hey foca' );
```

Comment:
Codacy has a fix for the issue: [Replace `·'hey·foca'·` with `"hey·foca"`](https://release.k8s.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=5795)
```suggestion
  console.log("hey foca");
```